src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -258,6 +258,7 @@
     <xsd:attribute name="id" type="xsd:string" />
     <xsd:attribute name="key" type="xsd:string" />
     <xsd:attribute name="on-invalid" type="invalid_sequence" />
+    <xsd:attribute name="trim-value" type="xsd:boolean" />
```

Comment:
looks like we use just "boolean" elsewhere
can you also fix "exclude-self" which has the same issue (in this PR) + use XmlUtils when accessing it?

Reply:
That's fixed for both :slightly_smiling_face: